src/Sulu/Bundle/AudienceTargetingBundle/Resources/config/services.xml

```diff
@@ -88,7 +88,7 @@
         </service>
         <!--CONTENT TYPES-->
         <service id="sulu_audience_targeting.content.type.audience_targeting_groups"
-                 class="Sulu\Bundle\AudienceTargetingBundle\Content\Types\AudienceTargetingGroups">
+                 class="Sulu\Bundle\AudienceTargetingBundle\Content\Types\AudienceTargetingGroups" public="true">
```

Comment:
all content-types will be loaded by id in class `Sulu\Component\Content\ContentTypeManager`